atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -123,6 +135,19 @@ private boolean isInitializedInternal() {
                     && txManager.getCleaner().isInitialized()
                     && initializationPrerequisite.get();
         }
+
+        private void runCallback(Callback callBack) {
```

Comment:
You're passing in `callBack` instead of using the field `callback`. Ugly 👎

Reply:
Oops, this is an artefact from previous version 👎